code-reviews/recipes/Bash.md

```diff
@@ -2,15 +2,17 @@
 
 ## Style Guide
 
-[CSE](../../../CSE.md) developers follow [Google's Bash Style Guide](https://google.github.io/styleguide/shell.xml).
+[CSE](../../CSE.md) developers follow [Google's Bash Style Guide](https://google.github.io/styleguide/shell.xml).
 
 ## Linter
 
-[CSE](../../../CSE.md) projects must check Bash code with [shellcheck](https://github.com/koalaman/shellcheck) as part of the [CI process](../../../continuous-integration/readme.md).
+[CSE](../../CSE.md) projects must check Bash code with [shellcheck](https://github.com/koalaman/shellcheck) as part of the [CI process](../../continuous-integration/readme.md).
+
+## Build Validation
```

Comment:
When I read build validation, I think that the section will be about how to compile a build for this language. It doesn't necessarily convey that is about linting. or maybe it is not, it just happens that the current content is only linting.
It would be helpful to add a mini description for this section to describe what should go there for anyone writing new recipes?

Reply:
Interesting... 

For AzDO build validation is defined as below.  I thought this was common terminology but perhaps it is not.
https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops

Where do you suggest this wording would go?  

I am thinking that someone that writes a new recipe would first look at the ones that already have this and follow suit, but perhaps that can't be assumed... I am lost as to where to write it though.

Reply:
Interesting.... I thought this was common terminology, perhaps it is just an AzDO thing
https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops

I was assuming anyone writing a new recipe would perhaps look at the others and get the gist, but perhaps we should clarify... I am at a bit of a loss though as to where to put it.  Do you have a suggestion? and what it should say?

Reply:
I think it is fine to have linting as part of the build but I hope we won't just add linting to build validation and rather add how to validate a build by compiling the language specific code (and running tests). The other sections also moslty include validating linting.

Reply:
maybe you can add a TODO to have language specific build validation in this section.

Reply:
To me build validation is really the process of automatically validating as much as possible of the code that is being PR-ed into the repo. For example, in the project we are running at the moment, we run StyleCop, FxCop, Unit tests and there is even a manual step to trigger an E2E test pipeline. Anything that aids validating the code before we have to manually look at it goes in here in my opinion.